README.md

```diff
@@ -45,4 +45,114 @@ sanitized_promise.then(function(sanitized_string) {
 
 ## Contributing
 
+### Commits
```

Comment:
At the top of this file, there are some Travis badges with `build:passing`. I'd replace those with badges from GitHub Actions (you can generate those from the checks tab)